api/services/elastigroup/azure/scaleSetVms/paths/group-deployments.yaml

```diff
@@ -6,8 +6,31 @@ get:
   tags:
     - Elastigroup Azure Scale Set VMs
   parameters:
-    - $ref: "../../../../../commons/parameters/accountId.yaml"
```

Comment:
@vlad666spot this part is already addressed in the master, please make sure you are aligned (no need to re-update)